ui/plugins/eu.esdihumboldt.hale.ui.templates/src/eu/esdihumboldt/hale/ui/templates/webtemplates/WebTemplateLoader.groovy

```diff
@@ -69,12 +65,14 @@ class WebTemplateLoader {
 			String password = System.getProperty("http.proxyPassword");
 			boolean useProxyAuth = userName != null && !userName.isEmpty();
 
-			if(useProxyAuth){
-
-				Credentials cred = new UsernamePasswordCredentials(userName, password);
-
-				builder.getClient().getCredentialsProvider().setCredentials(new AuthScope(proxyAddress.getHostName(),proxyAddress.getPort()),cred)
-			}
+			// TODO: Fix this
```

Comment:
What's the issue here?

Reply:
here it says:

```
Groovy:[Static type checking] - Cannot find matching method org.apache.http.client.HttpClient#getCredentialsProvider(). Please check if the declared type is correct and if the method exists.
```

I tried to find a replacement method but couldn't at that time so added a TODO. I will research it a bit more and update.

Reply:
Updated the code to use `Proxy-Authorization`.